tests/codegen.rs

```diff
@@ -96,7 +96,6 @@ async fn generated_code_is_fresh() {
     // https://hg.mozilla.org/projects/nss/file/tip/lib/certdb/genname.c such that
     // webpki-roots implements the same policy in this respect as the Mozilla root program.
     let mut imposed_constraints = HashMap::<Vec<u8>, Vec<u8>>::default();
-    imposed_constraints.insert(concat(ANSSI_SUBJECT_DN), concat(ANSSI_NAME_CONSTRAINTS));
```

Comment:
FWIW, I kept this around because it's still there in the upstream source in `genname.c`, so on the off chance that a certificate with the `ANSSI_SUBJECT_DN` ever makes it back in, this should apply to it.

Not sure that's a strong argument for keeping it, but it's why I kept it around in #34.